imports/plugins/core/orders/server/no-meteor/mutations/placeOrder.js

```diff
@@ -488,7 +495,31 @@ export default async function placeOrder(context, input) {
     finalGroup.itemIds = finalGroup.items.map((item) => item._id);
     finalGroup.totalItemQuantity = finalGroup.items.reduce((sum, item) => sum + item.quantity, 0);
 
-    finalGroup.invoice = getInvoiceForFulfillmentGroup(finalGroup, discountTotal, currencyCode);
+    // Get surcharges to apply to group, if applicable
+    let groupSurcharges = [];
+    const commonOrder = await xformOrderGroupToCommonOrder({
+      billingAddress,
+      cartId,
+      collections,
+      currencyCode,
+      group: finalGroup,
+      orderId,
+      discountTotal
+    });
+
+    for (const func of getFunctionsOfType("getSurcharges")) {
+      const appliedSurcharges = await func(context, { commonOrder }); // eslint-disable-line
+      appliedSurcharges.forEach((appliedSurcharge) => {
```

Comment:
Can you set `appliedSurcharge.fulfillmentGroupId` before pushing? It looks like the schema already allows it, and it would be good to be able to see which group each is for when looking at the array on `order`